content/docs/Konveyor/Kai/IDESettings/proc_configuring-developer-lightspeed-ide-settings.md

```
@@ -0,0 +1,57 @@
+| Demo mode | Enable to run Konveyor AI in demo mode that uses the LLM responses saved in the `cache` directory for analysis. |
+| Solution Server:URL | Edit Solution Server configurion.                      
+| Debug:Webview | Enable debug level logging for Webview message handling in VS Code.|
+
+Solution server configuration:
+* “enabled”: Enter a boolean value. Set `true` for connecting the Solution Server client (Konveyor extension) to the Solution Server. <br>
+                  
+* “url”: Configure the URL of the Solution Server end point.  <br>
+ 
+* “auth”: The authentication settings allows you to configure a list of options to authenticate to the Solution Server.  <br>
+
+    * "enabled": Set to `true` to enable authentication. If you enable authentication, then you must configure the Solution Server realm. <br> 
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)